tests/konflux-demo/konflux-demo.go

```diff
@@ -118,6 +118,7 @@ var _ = framework.KonfluxDemoSuiteDescribe(Label(devEnvTestLabel), func() {
 					if err != nil {
 						Expect(err.Error()).To(ContainSubstring("Reference does not exist"))
 					}
+					Expect(build.CleanupWebhooks(fw, componentRepositoryName)).ShouldNot(HaveOccurred())
```

Comment:
```suggestion
					Expect(build.CleanupWebhooks(fw, componentRepositoryName)).Should(Succeed())
```

Reply:
this assertion exists multiple places excluding the PR changes as well, I think it would be good to address in a separate PR and change in all the places.